packages/hyperion-react-testapp/src/AutoLoggingWrapper.ts

```diff
@@ -47,18 +47,19 @@ export function init() {
         {
           eventName: 'click',
           cacheElementReactInfo: true,
+          eventFilter: (domEvent: MouseEvent) => domEvent.isTrusted
         },
         {
           eventName: 'keydown',
           cacheElementReactInfo: true,
           interactableElementsOnly: true,
-          eventFilter: (domEvent: Event) => domEvent instanceof KeyboardEvent && domEvent?.code === 'Enter',
+          eventFilter: (domEvent: KeyboardEvent) => domEvent?.code === 'Enter',
```

Comment:
do you still need to specify the `KeyboardEvent`, or will it infer it correctly?

Reply:
Nope it's not needed, it infers correctly which is nice.
<img width="645" alt="image" src="https://user-images.githubusercontent.com/6064408/228565146-21cb2e36-5187-43c8-bccd-21904bc1d8d8.png">